vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss

```diff
@@ -0,0 +1,301 @@
+//     </div>
+//     <div class="us-ct-row__key-details">
+//       <button class="us-ct-row__key-details-toggle us-ct-row__key-details-toggle-open">View key details <svg role="presentation" class="us-ct-row__key-details-chevron"><use xlink:href="/images/icons.svg#icon-chevron-right"></use></svg></button>
+//       <div class="us-ct-row__key-details-info">
+//         <div class="us-ct-row__key-details-col">
+//           <strong>Card details</strong>
+//           <ul class="us-ct-row__key-details-ul">
+//             <li>£20 Amazon gift certificate exclusively through uSwitch (terms apply)</li>
+//             <li>0.5% cashback and no foreign transaction fees</li>
+//             <li>Credit limit increase possible after four months</li>
+//           </ul>
+//         </div>
+//         <div class="us-ct-row__key-details-col">
+//           <strong>To get this card you must at least:</strong>
+//           <ul class="us-ct-row__criteria us-ct-row__key-details-ul">
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>be 18 or older</li>
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>have a permanent UK address</li>
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>have a UK current bank or building society account</li>
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>not be registered bankrupt in past 18 months or have proceedings</li>
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>have no CCJs (past 12 months)</li>
+//           </ul>
+//           <p class="us-ct-row__criteria-additional">Additional criteria for acceptance from the lender may apply.</p>
+//         </div>
+//       </div>
+//     </div>
+//   </div>
+
+.us-ct-row {
+  position: relative;
+  display: block;
+  margin-bottom: 15px;
+  color: $c-navy;
+  background: #fff;
+  border: 1px solid $c-keylinegrey;
+  border-bottom-width: 5px;
+  border-radius: 5px;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &__title {
+    display: none;
+    width: 100%;
+    padding: 15px;
+    border-bottom: 1px solid $c-keylinegrey;
+  }
+
+  &__title-text {
+    margin: 0 15px 0 0;
+    font-weight: bold;
+  }
+
+  &__product {
+    position: relative;
```

Comment:
Is this needed? Why do we set position here?

Reply:
It's needed for some responsive functionality on tablets (absolute positioning on a child element)